haterBackEnd/hater/urls.py

```diff
@@ -14,8 +14,23 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
+from rest_framework import routers
+from hater_app.views import UserViewSet, User_profileViewSet, hatesViewSet, CriticismViewSet, DislikeViewSet, FollowerViewSet
+
+router = routers.DefaultRouter()
+# old haters profile viewset
+# router.register(r'Haters', HatersViewSet)
+router.register(r'user', UserViewSet)
```

Comment:
it's dangerous to have a route that spits out all users in our DB but you can keep this if u want just to protect the route so only people with admin privileges can view this route